### ISSUE TITLE
Create an email system

### ISSUE THREAD

**Issue by SundarakrishnanN** [IEEE-RVCE/GateKeeper#20]
create an email function that takes in parameters and sends any sort of automated email

**Comment by SundarakrishnanN** [comment]
> Please consider the springboot java mail documentation.
> 
> Additionally, for testing u will need to configure an email sending service. Gmail may work (u will need to create an app-specific password)

Sure will do the same

**Comment by Prajwalprakash3722** [comment]
I was thinking more of like AWS SES instead of gmail

**Comment by chrisvrose** [comment]
For local testing, Gmail should suffice, but SES would work well too.

**Comment by SundarakrishnanN** [comment]
SES is free for one year after which we have to pay,so if we choose SES,weve to switch accounts every year

**Comment by SundarakrishnanN** [comment]
> For local testing, Gmail should suffice, but SES would work well too.

yes,lets stick to gmail for now,anyways we can easily switch later on

**Comment by Prajwalprakash3722** [comment]
For local testing, it's better to use `Mailhog` to avoid Gmail marking your emails as spam.

**Comment by chrisvrose** [comment]
Works as well. 

~~(Although, word of warning: please don't send people billions of emails)~~

**Comment by SundarakrishnanN** [comment]
@ChiragJS 
Scenarios where u mail:
Once someone creates a request,mail the set of people who have to take action.
Mail requester saying Event Created
Everytime someone approves,just mail event requester to let them know about it..
While mailing make sure "unsubscribe" is false.. Add some dynamic content.

When an account is created:
Mail them with the password and strongly advice them to change password..
We will be adding option to edit password if uk current password.
